huntsman/tests/conftest.py

```diff
@@ -52,3 +57,81 @@ def db():
 @pytest.fixture
 def data_dir():
     return '{}/pocs/tests/data'.format(os.getenv('POCS'))
+
+
+def end_process(proc):
```

Comment:
Maybe a docstring explaining this deeply nest kill structure. :)

Reply:
Did have a few problems with these processes being left hanging after running tests....